fearless/Common/Network/GitHubOperationFactory.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import RobinHood
+import IrohaCrypto
+
+protocol GitHubOperationFactoryProtocol {
+    func fetchPhishingListOperation(_ url: URL) -> BaseOperation<[PhishingItem]>
+}
+
+class GitHubOperationFactory: GitHubOperationFactoryProtocol {
+    func fetchPhishingListOperation(_ url: URL) -> BaseOperation<[PhishingItem]> {
+        let requestFactory = BlockNetworkRequestFactory {
+            var request = URLRequest(url: url)
+            request.setValue(HttpContentType.json.rawValue,
+                             forHTTPHeaderField: HttpHeaderKey.contentType.rawValue)
+            request.httpMethod = HttpMethod.get.rawValue
+            return request
+        }
+
+        let resultFactory = AnyNetworkResultFactory<[PhishingItem]> { data in
+            var phishingItems: [PhishingItem] = []
+
+            if let json = try JSONSerialization.jsonObject(with: data,
+                                                           options: [.mutableContainers]) as? [String: AnyObject] {
+                for (key, value) in json {
```

Comment:
It is not so critical but why not to use functional style compactMap/flatMap/map here. The main benefit is that we are sure that we put an item at most once to the list in the loop.